en/micro-integrator/docs/references/mediators/property-reference/generic-Properties.md

```diff
@@ -761,13 +761,141 @@ Generic properties allow you to configure messages as they're processed by the 
 </tr>
 <tr class="even">
 <td><p><strong>Example</strong></p></td>
-<td><div class="content-wrapper">
+<td>
+	<div class="content-wrapper">
 <div class="code panel pdl" style="border-width: 1px;">
 <div class="codeContent panelContent pdl">
-<div class="sourceCode" id="cb1" data-syntaxhighlighter-params="brush: java; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: java; gutter: false; theme: Confluence"><pre class="sourceCode java"><code class="sourceCode java"><span id="cb1-1"><a href="#cb1-1"></a>&lt;property name=<span class="st">&quot;QUOTE_STRING_IN_PAYLOAD_FACTORY_JSON&quot;</span> value=<span class="st">&quot;true&quot;</span>/&gt; </span></code></pre></div>
+<div class="sourceCode" id="cb1" data-syntaxhighlighter-params="brush: java; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: java; gutter: false; theme: Confluence"><pre class="sourceCode java"><code class="sourceCode java"><span id="cb1-1"><a href="#cb1-1"></a>&lt;property name=<span class="st">&quot;QUOTE_STRING_IN_PAYLOAD_FACTORY_JSON&quot;</span> value=<span class="st">&quot;true&quot;</span>/&gt; </span></code></pre>
 </div>
 </div>
-</div></td>
+</div>
+</div>
+</td>
 </tr>
 </tbody>
 </table>
+
+## RabbitMQ Properties
+
+The following generic properties can be used in the [Property mediator](../../../references/mediators/property-Mediator.md) and the [Property Group mediator](../../../references/mediators/property-Group-Mediator.md) for RabbitMQ use cases.
+
+### SET_ROLLBACK_ONLY
+
+<table>
+	<tr>
+		<th>
+			Parameter
+		</th>
+		<th>
+			Description
+		</th>
+	</tr>
+	<tr>
+		<td>
+			Name
+		</td>
+		<td>
+			SET_ROLLBACK_ONLY
+		</td>
+	</tr>
+	<tr>
+		<td>
+			Possible Values
+		</td>
+		<td>
+			true/false
+		</td>
+	</tr>
+	<tr>
+		<td>
+			Scope
+		</td>
+		<td>
+			default
+		</td>
+	</tr>
+	<tr>
+		<td>
+			Description
+		</td>
+		<td>
+			When a message is read from an inbound (RabbitMQ) message queue via an <b>Inbound Endpoint</b>, it will be sent to a service running in the backend. If a failure occurs, the transaction will roll back avoiding message loss.
```

Comment:
```suggestion
			When a message is read from an inbound (RabbitMQ) message queue via an <b>Inbound Endpoint</b>, it will be sent to a service running in the backend. If a failure occurs, the Micro Integrator will do a <b>basicReject</b> with the requeue flag set to 'false'. In that case, the user must configure a Dead Letter Exchange to avoid losing messages. The same concept could be used to control the number of retries and delay messages. Please refer to the documentation for more details.